packages/components/src/components/badge/badge.scss

```
@@ -0,0 +1,141 @@
+@use "sass:map";
+@use "@db-ui/foundations/build/scss/init" as *;
+@use "@db-ui/foundations/build/scss/variables" as *;
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+@use "@db-ui/foundations/build/scss/color-placeholder" as *;
+@use "@db-ui/foundations/build/scss/color/color-variants" as *;
+@use "@db-ui/foundations/build/scss/helpers/component" as *;
+@use "@db-ui/foundations/build/scss/body-styles" as *;
+@use "../../styles/button-components";
+
+%tag-colors {
+	// TODO: Color from zeplin does not exist is it neutral-4??
+	@extend %db-bg-neutral-4;
+
+	&[data-emphasis="strong"] {
+		@extend %db-neutral-component;
+	}
+
+	@each $name, $colors in $component-variants {
+		&[data-variant="#{$name}"] {
+			@if ($name == "critical") {
+				@extend %db-bg-critical;
+
+				&[data-emphasis="strong"] {
+					@extend %db-critical-component;
+				}
+			} @else if ($name == "informational") {
+				@extend %db-bg-informational;
+
+				&[data-emphasis="strong"] {
+					@extend %db-informational-component;
+				}
+			} @else if ($name == "warning") {
+				@extend %db-bg-warning;
+
+				&[data-emphasis="strong"] {
+					@extend %db-warning-component;
+				}
+			} @else if ($name == "successful") {
+				@extend %db-bg-successful;
+
+				&[data-emphasis="strong"] {
+					@extend %db-successful-component;
+				}
+			}
+
+			&:not(&[data-emphasis="strong"]) {
+				--db-current-border-color: #{map.get($colors, "border")};
+			}
+		}
+	}
+}
+
+%absolute-badge {
+	position: absolute;
+
+	transform: translate(var(--badge-transform-x), var(--badge-transform-y));
+
+	&[data-placement$="right"] {
+		--badge-transform-x: 50%;
+		inset-inline-end: 0;
+	}
+
+	&[data-placement$="left"] {
+		--badge-transform-x: -50%;
+		inset-inline-start: 0;
+	}
+
+	&[data-placement*="top"] {
+		--badge-transform-y: -50%;
+		inset-block-start: 0;
+	}
+
+	&[data-placement*="center"] {
+		--badge-transform-y: -50%;
+		inset-block-start: 50%;
+	}
+
+	&[data-placement*="bottom"] {
+		--badge-transform-y: 50%;
+		inset-block-end: 0;
+	}
+}
+
+.db-badge {
+	--badge-size: #{$db-spacing-fixed-sm};
+	@extend %tag-colors;
+	@extend %component-border;
+	@extend %db-body-sm;
+	@extend %default-button;
+
+	display: flex;
+	border-radius: 42px; // extreme radius
+	padding-inline: $db-spacing-fixed-xs;
+	block-size: fit-content;
+	pointer-events: none;
+
+	&:empty,
+	> span:empty,
+	&[data-no-text="true"] {
+		block-size: var(--badge-size);
+		inline-size: var(--badge-size);
+		padding: 0;
+	}
+
+	&:has(.db-icon) {
+		padding-block: $db-spacing-fixed-2xs;
+		padding-inline: $db-spacing-fixed-2xs;
```

```suggestion
		padding: $db-spacing-fixed-2xs;
```

Doesn't this mean the general `padding`?

---

fixed